puente-etl/utils/constants.py

```diff
@@ -0,0 +1,104 @@
+class ColumnOrder:
+    # Enforce column order for relational table outputs
+    FORM_SPECIFICATIONS = [
+        'custom_form_id',
+        'custom_form_organizations',
+        'custom_form_class',
+        'custom_form_type_of_form',
+        'custom_form_name',
+        'custom_form_description',
+        'custom_form_header',
+        #
+        # Excluding these for now for readability:
+        #
+        # 'custom_form_updated_at',
+        # 'custom_form_created_at',
+        # 'custom_form_custom_form',
+        # 'custom_form_workflows',
+        # 'custom_form_active',
+        'question_id',
+        'question_label',
+        'question_text',
+        'question_field_type',
+        'question_formik_key',
+        'question_value',
+        #
+        # Excluding these for now for readability:
+        #
+        # 'question_active',
+        # 'question_number_questions_to_repeat',
+        # 'question_side_label',
+        # 'question_validation',
+        'answer_id',
+        'answer_label',
+        'answer_value',
+        'answer_text',
+        'answer_text_key',
+        'answer_text_question'
+    ]
+
+    FORM_RESULTS = [
+        'form_result_id',
+        'form_result_surveying_organization',
+        'form_result_surveying_user',
+        'custom_form_id',
+        'form_result_response_id',
+        'question_title',
+        'question_answer',
+        'question_type',
+        #
+        # Excluding these for now for readability:
+        #
+        # 'custom_form_name',
+        # 'custom_form_description',
+        # 'form_result_created_at',
```

Comment:
need 

- `form_result_created_at`

Reply:
This one does not naturally fit into the aggregated view (one org, one form specification, many form results), but if you need the denormalized form_specifications table you could add it back. I'll exclude for now.